### PR TITLE
✨  [amp-render] validator changes

### PR DESCRIPTION
**Pull request by dmanek**
Partial for #32477

### CODE REVIEW

File: extensions/amp-render/validator-amp-render.protoascii
Comment (honeybadgerdontcare) [inline extensions/amp-render/validator-amp-render.protoascii:39]: please alphabetize by attribute name

File: extensions/amp-render/validator-amp-render.protoascii
Comment (honeybadgerdontcare) [inline extensions/amp-render/validator-amp-render.protoascii:46]: note there is no `src` attribute listed here. also why valid test examples are failing in the validator test file.

File: extensions/amp-render/validator-amp-render.protoascii
Comment (honeybadgerdontcare) [inline extensions/amp-render/validator-amp-render.protoascii:49]: please uncomment if we're expecting this url to be real. is there a doc PR doing that?

File: extensions/amp-render/validator-amp-render.protoascii
Comment (honeybadgerdontcare) [inline extensions/amp-render/validator-amp-render.protoascii:45]: is this expected to be an URL? should we use `value_url` here to enforce protocols and whether relative is allowed? I'm guessing relative URLs should not be allowed in this case.

**Review by honeybadgerdontcare — CHANGES_REQUESTED** [review]
The I2I linked does not show any approvers approving it. Has this received approval to move forward?

File: extensions/amp-render/validator-amp-render.protoascii
Comment (dmanek) [reply extensions/amp-render/validator-amp-render.protoascii:57]: Ah my oversight! Added the `src` attribute which fixes the tests.

File: extensions/amp-render/validator-amp-render.protoascii
Comment (dmanek) [reply extensions/amp-render/validator-amp-render.protoascii:56]: Added this in the `src` attribute. It can be an absolute or relative URL, amp-script or amp-bind.

**Comment by dmanek** [comment]
Yes, this I2I was approved in the design review on 2-10-21.

File: extensions/amp-render/1.0/test/validator-amp-render.out
Comment (micajuine-ho) [inline extensions/amp-render/1.0/test/validator-amp-render.out:143]: nit: newline here I think

**Review by micajuine-ho — APPROVED** [review]
LGTM

File: extensions/amp-render/validator-amp-render.protoascii
Comment (honeybadgerdontcare) [inline extensions/amp-render/validator-amp-render.protoascii:47]: please alphabetize

File: extensions/amp-render/validator-amp-render.protoascii
Comment (honeybadgerdontcare) [inline extensions/amp-render/validator-amp-render.protoascii:50]: `allow_relative` defaults to true, so it's not necessary here

**Review by honeybadgerdontcare — APPROVED** [review]

File: extensions/amp-render/1.0/test/validator-amp-render.out
Comment (dmanek) [reply extensions/amp-render/1.0/test/validator-amp-render.out:143]: This file is autogenerated, so don't want to mess with the output.

**Pull request merged**
